.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish Package to npmjs
```

Comment:
Is it possible to have this workflow not happen if the other workflow (build and test) fails?

Reply:
Good question. Currently the test workflow doesn't really do anything different so it doesn't matter but that's a good idea.

Reply:
done